pages/wizard/product_selection.py

```diff
@@ -39,3 +41,15 @@ def click_next(self):
     # there is no back button on this page:
     def click_back(self):
         return None
+
+    @property
+    def is_the_current_page(self):
+        WebDriverWait(self.selenium, self.timeout).until(
+            EC.visibility_of(self.rhev_checkbox),
+            "Expected element not present on Select Products Page")
+        return True
+
+    def cancel_deployment(self):
```

Comment:
I think there was generic cancel, well I think you select it in here...are you wrapping this up because here it goes to different place?